### ISSUE TITLE
Push: problem with EventBusFactory.getDefault() after upgrading from 5.1 to 5.2

### ISSUE THREAD

**Issue by fkjaekel** [primefaces/primefaces#678]
Hi,

Jeanfrancois asked me to file an issue here. https://goo.gl/dDBHzd

In order to upgrade from 5.1 to 5.2, I upgraded atmosphere-runtime from 2.2.8 to 2.3.4.

I have two webapps on the same server that use Atmosphere. (TomEE 1.7.2) 

webapp1 has a stateless EJB timer that is used to periodically push some notifications.

When only webapp1 is deployed, the timer works fine, but after I deploy webapp2, EventBusFactory.getDefault() starts to return null.

Both webapps pack atmosphere-runtime inside WEB-INF/lib. This issue doesn't happens with version 2.2.8.

If I move atmosphere-runtime 2.3.4 to the server lib it works fine with both webapps deployed.

I tried the CDI workaround suggested, but I still get a NPE. I implemented it like this:

``` javascript
@ApplicationScoped
public class EventBusProducer implements Serializable
{
    private static final long   serialVersionUID    = 1L;

    @Produces
    public EventBus createEventBus()
    {
        return EventBusFactory.getDefault().eventBus();
    }
}
```

Thanks

**Comment by tandraschko** [comment]
Could you please try it with the current snapshot? we upgraded atmosphere, maybe it's gone.

**Comment by kukel** [comment]
Is this still a problem? I've seen closed issues, but 6.0.2 users mentions this same problem

**Comment by fkjaekel** [comment]
I didn't tested push with 6.0 yet. Currently I'm using 5.3.15, and after adding `<load-on-startup>1</load-on-startup>` to the PushServlet declaration I didn't have this issue anymore.
